programs/stake/src/stake_state.rs

```diff
@@ -362,43 +362,57 @@ impl Authorized {
     }
 }
 
+/// captures a rewards round as lamports to be awarded
+///  and the total points over which those lamports
+///  are to be distributed
+//  basically read as rewards/points, but in integers instead of as an f64
+pub struct PointValue {
```

Comment:
nits: I'd like this to be named like `InflationRound` or something to be neutral to both of its members `rewards` and `points`.

Reply:
because the previous code used "point_value" as the name of an f64, I'm inclined to leave this as is because it matches previous meaning:
`PointValue: point_value`

There's more to an InflationRound than validator points...